roles/foreman/tasks/main.yaml

```diff
@@ -87,6 +87,20 @@
       DYNFLOW_REDIS_URL: "redis://localhost:6379/6"
       REDIS_PROVIDER: "DYNFLOW_REDIS_URL"
     command: "/usr/libexec/foreman/sidekiq-selinux -e production -r /usr/share/foreman/extras/dynflow-sidekiq.rb -C /etc/foreman/dynflow/%i.yml"
```

Comment:
I wonder if the `sidekiq-selinux` start up command will be needed inside a container.

Reply:
Maybe. Maybe not. Probably not.